puncto-back/src/exceptions/UnauthorizedException.ts

```diff
@@ -0,0 +1,10 @@
+import { BaseException } from './BaseException';
+
+export class UnauthorizedException extends BaseException {
+    constructor(message: string) {
+        super();
+        this.message = message;
+        this.type = 'Unauthorized';
+        this.statusCode = 401;
+      }
```

Comment:
@gabrielchaves7 ve o que vc acha desse esquema de error handling também

Pensei que podemos separar os tipos de erro só pelo codigo (unauthorized, database, not found, etc) mesmo, ao inves de criar um específico pra cada rota

Reply:
Esta ótimo